src/test/java/uk/gov/ons/fwmt/job_service/service/impl/totalmobile/TMJobConverterServiceImplTest.java

```diff
@@ -193,6 +199,38 @@ public void updateJobFromIngest() {
     assertNotNull(result);
   }
 
+  @Test
+  public void checkIfAddressLineIsNotBlank() {
+    //Given
+    List<String> testAddressLines = new ArrayList<>();
+    String testLine = "testLine";
+
+    List<String> expectedArray = new ArrayList<>();
+    expectedArray.add("testLine");
+
+    //When
+    tmJobConverterService.addAddressLines(testAddressLines,testLine);
+
+    //Then
+    assertEquals(testAddressLines, expectedArray);
+  }
+
+  @Test
+  public void checkIfAddressLineIsBlank() {
+    //Given
+    List<String> testAddressLines = new ArrayList<>();
+    String testLine = "";
+
+    List<String> expectedArray = new ArrayList<>();
+    expectedArray.add("testLine");
+
+    //When
+    tmJobConverterService.addAddressLines(testAddressLines,testLine);
+
+    //Then
+    assertNotEquals(testAddressLines, expectedArray);
```

Comment:
expected result should be first.
asserting they are not equal I dont think is enough, Id add assertEquals(0, testAddressLines.size())